ha_vpn_over_interconnect_10GB_attach/main.tf

```diff
@@ -0,0 +1,421 @@
+# HA VPN over Cloud Interconnect 10GB VLAN attachments (Dedicated Interconnect)
+
+# [START cloudinterconnect_ha_vpn_over_interconnect_10gb_attachments]
+
+# Create all resources in the same region, which you can specify at the provider-level
+ provider "google" {
+  project = "your-project"
```

Comment:
Please remove the "project" attribute from every resource where you have included it. Our instructions to users are to run the samples in Cloud Shell. This attribute isn't needed in Cloud Shell and causes a speed bump for users to update the text or delete the attribute before they can successfully run the sample.

Reply:
Would it be ok if I added a comment here instead of removing? With Interconnect, there's the option of creating your attachments in a different project than your connection. So in this case, you would be specifying a different project than your Interconnect connection project (which are specified via URLs on lines 63 and 77). 

Adds complexity but more flexibility to the sample.

Reply:
OK, let's keep it but comment it out so that copy/paste/run in Cloud Shell works.

Reply:
Moved project into a comment, added comments around using it